packages/deployment/upgrade-test/Readme.md

```diff
@@ -26,6 +26,19 @@ make build
 make run
 ```
 
+This will start a container with tmux, with the first window `0` being chain logs `agd start` and the second and current window `1` being a bash shell. You can navigate using `bind-key+B N` (assuming `bind-key` is CTRL/CMD) and N is the window. For more shortcuts see [tmux shortcuts & cheatsheet](https://gist.github.com/MohamedAlaa/2961058#list-all-shortcuts).
+
+The container and chain will halt once you detach from the session.
+
+### Using tmux control mode
+
+If you use [iTerm you can use tmux with native integration](https://iterm2.com/documentation-tmux-integration.html), called control mode, which will make your tmux session appear as a physical window. Pass `TMUX_CC=1`:
+
+```shell
+TMUX_CC=1 make run
```

Comment:
what happens in other terminals when this is enabled?

Reply:
You don't want that. Control mode makes it so tmux outputs raw commands that the terminal can interpret and render natively, so you'll get a stream of text on top of the usual tmux window.

Reply:
got it

Reply:
Updated readme with a warning to not use if not supported.